src/m365/planner/commands/task/task-get.ts

```diff
@@ -30,15 +31,29 @@ class PlannerTaskGetCommand extends GraphCommand {
       },
       responseType: 'json'
     };
+    let taskItem: Task;
 
     request
       .get(requestOptions)
-      .then((res: any): void => {
-        logger.log(res);
+      .then((res: any): Promise<BetaTask> => {
+        taskItem = res;
+        requestOptions.url = `${this.resource}/beta/planner/tasks/${encodeURIComponent(args.options.id)}`;
+        return request.get(requestOptions);
+      })
+      .then((betaTaskItem: BetaTask): void => {
+        logger.log(this.mergeTaskPriority(taskItem, betaTaskItem));
         cb();
       }, (err: any): void => this.handleRejectedODataJsonPromise(err, logger, cb));
   }
 
+  private mergeTaskPriority(taskItem: Task, betaTaskItem: BetaTask): BetaTask {
+    if (betaTaskItem) {
```

Comment:
I don't think this will work here. We should check the API response for this endpoint, but my gut feeling says that if retrieving the task fails, it will lead to a 404 which will reject the Promise, so when we end up here, we can assume that we have retrieve the task, so we don't need this if and could merge the priority already in line 44 without an extra method.

Reply:
Will check it